gst/nnstreamer/tensor_decoder/tensordec.c

```diff
@@ -542,6 +548,40 @@ gst_tensordec_get_property (GObject * object, guint prop_id,
       PROP_READ_OPTION (7);
       PROP_READ_OPTION (8);
       PROP_READ_OPTION (9);
+    case PROP_SUBPLUGINS:
+    {
+      subplugin_info_s sinfo;
+      guint total;
+
+      total = nnsconf_get_subplugin_info (NNSCONF_PATH_DECODERS, &sinfo);
+
+      if (total > 0) {
+        GString *subplugins;
+        const gchar *prefix_str;
+        guint i, prefix, len;
+
+        subplugins = g_string_new (NULL);
+
+        prefix_str = nnsconf_get_subplugin_name_prefix (NNSCONF_PATH_DECODERS);
+        prefix = strlen (prefix_str);
```

Comment:
strlen returns size_t. Is it safe to use guint here?

Reply:
@wooksong Thanks, I just updated the type (gsize)